tests/test_description.py

```diff
@@ -40,5 +40,12 @@ def test_lambda_content_with_captures():
     assert 'x == y' == e.value.description
 
 
+def test_multi_line_lambda_content_description_on_timeout():
+    with pytest.raises(ConditionTimeoutError) as e:
+        wait_at_most(ONE_HUNDRED_MILLISECONDS).until(lambda: 3 == 4 and \
+                                                             5 == 6)
+    assert '3 == 4 and \\ 5 == 6' == e.value.description
```

Comment:
Do we want the \\?

Reply:
yes, the whole point is to test lambda spanning on multiple lines and this was the easiest way to do that

Reply:
You're right, we don't need it